packages/cli/src/commands/upgrade/upgrade.js

```diff
@@ -214,7 +234,7 @@ async function upgrade(argv: Array<string>, ctx: ContextT, args: FlagsT) {
     return;
   }
 
-  const patch = await getPatch(currentVersion, newVersion, projectDir);
+  const patch = await getPatch(currentVersion, newVersion, projectDir, ctx);
```

Comment:
I think `projectDir` is also on `ctx` as `root` so might be better to stick to that one?

Reply:
Turned out `projectDir` was not used :D